src/mixins/filing-template-mixin.ts

```diff
@@ -167,7 +171,10 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         incorporationAgreement: {
           agreementType: this.getAgreementType
         }
-      }
+      },
+      // *** FUTURE: implement these:
+      changeofRegistration: undefined,
+      registration: undefined
```

Comment:
I may keep working on this today, but there remain some unknowns and this should be part of future tickets. Oh well, this whole PR is to try to advance things as much as possible for future firm corrections work.